app/helpers/blacklight_helper.rb

```diff
@@ -10,9 +10,11 @@ def document_partial_name(document, base_name = nil)
 
     display_type ||= document.display_type
 
+    display_type ||= document['eds_publication_type']
```

Comment:
We're overriding this method, so I'm not entirely sure what the best way to ensure that we get the display_type for the research starters. It's a different field than the display_type configuration in ArticleController.